modules/network-observability-flows-format.adoc

```diff
@@ -0,0 +1,246 @@
+// Automatically generated by 'hack/asciidoc-flows-gen.sh'. Do not edit.
+:_content-type: REFERENCE
+[id="network-observability-flows-format-specifications_{context}"]
+= Network Flows format specifications
+
+The document is organized in two main categories: _Labels_ and regular _Fields_. This distinction only matters when querying Loki. This is because _Labels_, unlike _Fields_, must be used in link:https://grafana.com/docs/loki/latest/logql/log_queries/#log-stream-selector[stream selectors].
+
+Outside of Loki context, for instance when reading this specification as a reference for the Kafka export feature, you must ignore this distinction about _Labels_ and _Fields_, and consider all of them at the same level in the JSON structure.
```

Comment:
```suggestion
Outside of Loki context, for instance when reading this specification as a reference for the Kafka export feature, you must ignore this distinction about _Labels_ and _Fields_, and consider all of them as _Fields_.
```

Reply:
@jotak wdyt about this suggestion? I know you already merged the generated JSON doc.

Reply:
I don't want the users to think that the json for Kafka is like:

```
[{
  Fields: {
    SrcK8s_Namespace: "...",
    // etc.
  }
}]
```

whereas it's:

```
[{
  SrcK8s_Namespace: "...",
  // etc.
}]
```

Maybe the previous formulation can be improved, but I would not mention `Fields` when we're not talking about Loki

Reply:
maybe just say: `you must ignore this distinction about _Labels_ and _Fields_, and consider all of them as regular fields.` ?

Reply:
yeah your last comment looks good to me, do you think it would be helpful to copy the FLOW JSON record as an example?

Reply:
I chatted with @memodi about this offline because I had the same query for Joel. I shared with him what Joel said to me, which is that it could be more relevant for a blog post or something in that vein, but in docs it's more "risky" because the instruction could vary depending on splunk/ES versions , support plans that users have, etc. many things that are not in our control and responsibility.